frontend/src/components/ScenarioBox.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <v-sheet
+    class="pa-4 resource-box mb-6"
+    elevation="0"
+    rounded
+    :id="id"
+  >
+    <h4 class="ma-2 section-subtitle">{{ title }}</h4>
+    <p>{{ description }}</p>
+
+    <v-row class="mt-4" dense>
+      <!-- PDF -->
+      <v-col cols="12" md="6">
+        <v-card
+          class="overflow-hidden"
+          style="border-radius: 6px; cursor: pointer;"
+          @click="openPdf(taskPdf, title)"
+        >
+          <iframe
+            :src="taskPdf + '#page=1&view=fitH&toolbar=0'"
+            :title="title + ' Task PDF'"
+            style="width: 100%; height: 250px; border: none;"
+          ></iframe>
```

Comment:
I'm a little hesitant on all of these iframes. But I assume that you evaluated hosting this content ourselves and chose embedded iframes for a reason...